docs/examples/fork-process-model/flask-gunicorn/README.rst

```diff
@@ -0,0 +1,13 @@
+Installation
+------------
+.. code-block:: sh
+
+    pip install -rrequirements.txt
+
+Run application
+---------------
+.. code-block:: sh
+
+    gunicorn app -c gunicorn.config.py
+
+Please make sure Jaeger is running locally and the default agent host and port are exposed.
```

Comment:
Don't think we should specifically call out Jaeger. If we have to, I'd say either suggest running OTLP  + Otel Collector or keep it generic.

Reply:
Yea, I meant using something like console exporter. Either console or OTLP is fine but personally I think we should always mention OTLP is docs and examples as we should be by default making it very obvious how to get end to end observability working with OpenTelemetry and that includes the collector.